Major drought maps

hcorson-dosch-usgs opened the pull request:
### Changes
* In the pipeline, tweak the sizing for the drought period maps and main inset map so that they are identical
* Use `v-for` to dynamically add the map images for each of the major droughts, using the `id` field in `scrollToDates` to source the images and build the alt text
* Place the dynamically added images and original map image within a new div with id `"inset-image-container"` and set the sticky positioning on that div, instead of the individual images
* Retain `position: sticky` for the default drought map, so that it scrolls up with the grid at the end of the timeline
* Set `position: absolute;` on the image class used for all drought-specific map images, so that they stack on top of the default map
* Add the class `hide` with `display: none;` to hide the drought-specific maps
* Use the scrollTo rectangles to trigger the `show` class for the map image that corresponds each drought when in that drought period
  * Note: currently the image just stacks on top of the original map, which means the shadow effect is doubled. I could change this by hiding the original image, but I actually kind of like it as a subtle way of highlighting that the map has changed
* Unrelated, but I also fixed the bug w/ the 2nd legend annotation on mobile by combining it with the first 

### Image example
![image](https://github.com/DOI-USGS/drought-timeline/assets/54007288/1684b75d-f47d-4755-aee5-cc3bf73604b3)


### To test
Pull changes to local clone and run `npm run serve`

aaarcher-usgs commented on src/components/DroughtHistory.vue:
```diff
@@ -435,7 +445,18 @@ export default {
+            scrollTrigger: {
+              markers: false,
+              trigger: `#${scrollIDFull}`,
+              start: "top 25%",
```
I played around a little bit with where these triggers sit. I think 10% works a bit better because right now at 25%, the grey is present for quite a while after the map and button go back to "no drought" status. I imagine we'll tweak all these triggers a bit though once we're in mobile testing. 

![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/b78384e2-e6b3-45e2-aa9b-e838b0d5e942)

aaarcher-usgs reviewed (approved):
This works really smoothly, Just how I imagined it! 

I added a comment about where scroll triggers are being placed for the "in drought" and "not in drought" parts of the timeline. I think that can be tabled until we're doing mobile testing, since we'll need to make sure triggers are working consistently across devices/OSs anyway. 

I do think that the points are a bit hard to see when they're "red" right now, not sure if we want to bring that to design studio or something. I think they're passable as-is for now, but something we may want to revisit.

hcorson-dosch-usgs commented:
> I added a comment about where scroll triggers are being placed for the "in drought" and "not in drought" parts of the timeline. I think that can be tabled until we're doing mobile testing, since we'll need to make sure triggers are working consistently across devices/OSs anyway.

Yes, we will definitely need to adjust those triggers - I added an issue (#27) 

> I do think that the points are a bit hard to see when they're "red" right now, not sure if we want to bring that to design studio or something. I think they're passable as-is for now, but something we may want to revisit.

Agreed -- might be worth demoing in design studio
